app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
+    }
+  });
+
+  $('.form-control').on('mouseenter mouseleave focus blur',function(e){
+    $('.form-group input').checkForm(e);
+  });
+
+  $('#no-sunet-submit').click(function(e) {
+    if ($('.form-group input').checkForm(e)){
+      $('form').submit();
+    }
+  });
+});
+
+$.fn.checkForm = function(e) {
+  var empty = true;
+
+  $(this).each(function() {
```

Comment:
So then what do we want? I guess I am confused about the requirements. 

#request_user_attributes_name is a non-required element in the .form-group, so I guess other non-required elements that are added would have to be treated the same.